gc/base/GCExtensionsBase.cpp

```diff
@@ -182,10 +182,12 @@ MM_GCExtensionsBase::initialize(MM_EnvironmentBase* env)
 	if (J9HookInitializeInterface(getPrivateHookInterface(), OMRPORTLIB, sizeof(privateHookInterface))) {
 		goto failed;
 	}
+	INITIALIZE_HOOKDUMP(privateHookInterface);
```

Comment:
Why is the `INITIALIZE_HOOKDUMP` macro needed?  Could you roll it into the `J9HookInitializeInterface()` call?  

It's easier for consumers of this code if they don't have to do anything special after calling `J9HookInitializeInterface` as they may forget / not know about the extra macro step.